pytest/tests/sanity/routing_table_sync.py

```diff
@@ -170,29 +170,13 @@ def adv_get_routing_table(n):
         var_b = {(x['key'][0], x['key'][1], x['nonce']) for x in var_b}
         return var_a, var_b
 
-    def adv_get_routing_table_new(n):
-        logger.info("getting routing tables")
-        var_a = n[0].json_rpc("adv_get_routing_table_new", {},
-                              timeout=60)["result"]["edges_info"]
-        var_b = n[1].json_rpc("adv_get_routing_table_new", {},
-                              timeout=60)["result"]["edges_info"]
-        var_a = {(x['key'][0], x['key'][1], x['nonce']) for x in var_a}
-        var_b = {(x['key'][0], x['key'][1], x['nonce']) for x in var_b}
-        return var_a, var_b
```

Comment:
Why do we remove this function? It seems that we removed part of the test. Is that intentional?

Reply:
We used to have copy of routing table in two places (`PeerManagerActor`, `RoutingTableActor`). After we merge this PR we will have only one place with routing table. So, the second method is no longer needed.